### PR TITLE
Add cd configuration on Github Actions

### PR DESCRIPTION
**Pull request by thebevrishot**
closes https://github.com/zcoinofficial/ztm/issues/92

### CODE REVIEW

**Comment by thebevrishot** [comment]
@ultimaweapon After investigate I think we need 1 - 2 days to create the action my idea is we could let it be duplicated for now and fix it later after first release.

File: .github/workflows/cd.yml [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+      uses: actions/upload-artifact@v1
+      with:
+        name: migration
+        path: migration
+  docker:
+    name: Build Docker Image
+    needs: build
+    runs-on: ubuntu-18.04
+    steps:
+    - name: Checkout
```
Comment (ultimaweapon) [inline .github/workflows/cd.yml]: Seems like we don't need this.

File: .github/workflows/cd.yml [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+      with:
+        name: release
+        path: release
+    - name: Fix File Permission
+      run: chmod +x release/Ztm.WebApi
+    - name: Login Docker
+      uses: azure/docker-login@v1
+      with:
+        login-server: docker.pkg.github.com
+        username: $GITHUB_ACTOR
```
Comment (ultimaweapon) [inline .github/workflows/cd.yml]: I think better to use `secrets` so it match with `password`.

File: .github/workflows/cd.yml (hunk 1 above)
Comment (thebevrishot) [reply .github/workflows/cd.yml]: I believe we need this because we have to access Dockerfile

File: .github/workflows/cd.yml (hunk 2 above)
Comment (thebevrishot) [reply .github/workflows/cd.yml]: Not sure what field from `secrets` you mean. I updated to `${{ github.repository }}` instead.

File: .github/workflows/cd.yml (hunk 1 above)
Comment (ultimaweapon) [reply .github/workflows/cd.yml]: I think we should create artifact for that Dockerfile instead.

**Pull request merged**